### ISSUE TITLE
Reports PDF returns "403 THIS ACTION IS UNAUTHORIZED."

### ISSUE THREAD

**Issue by fabianober** [InvoiceShelf/InvoiceShelf#48]
Hi, first of all: Amazing, that this project was kept alive!!!

When accessing `/admin/reports`, the iFrame that should render the respective PDF showign the report of "Sales", "Expenses", "Taxes" etc. shows `403 THIS ACTION IS UNAUTHORIZED`

<img width="1512" src="https://github.com/InvoiceShelf/InvoiceShelf/assets/61794211/b1499730-d763-4574-a603-a2a33929bd7d">

Even after logout and login, the frame still shows 403.

**Comment by fabianober** [comment]
Same happens for `/reports/expenses/:id/download-receipt`, also shows 
`403 | THIS ACTION IS UNAUTHORIZED.`

**Comment by fabianober** [comment]
in `/config/abilities.php` it says the following from line 358 on:

```
// Financial Reports
        [
            'name' => 'view financial reports',
            'ability' => 'view-financial-reports',
            'model' => null,
        ],
```

Probably, the `model` must be set? But i am not sure to what.

**Comment by gdarko** [comment]
Hey @fabianober  - This issue will be fixed in version v1.2.0 which is not released yet. Currently the issue is fixed in the master branch of the app. If you can pull it that would be great.

**Comment by mona-chen** [comment]
@gdarko Please can you point me to your fix for this -> i tried upgrading my crater installation which i already made some changes to, but there where lot of merge conflicts and after resolving them the app wasnt working as it should, so i decided to cherry pick the updates, i've upgraded all dependencies and made neccesary changes but i'm stuck with this error, kindly help.

**Comment by gdarko** [comment]
@mona-chen - The issue is related to Bouncer.

https://github.com/InvoiceShelf/InvoiceShelf/commit/0d006846d517090a76874cd7b6b5eb0061f71c8e

Closing this as fixed in 1.2.0+
